src/components/Footer/Footer.js

```diff
@@ -23,6 +24,28 @@ export const Footer = () => {
   const { theme } = useContext(ThemeContext);
   const { t } = useTranslation();
   const { isDarkTheme } = useContext(ThemeContext);
+  const LINKS = [
```

Comment:
@SylwiaHaracz świetnie o to chodziło, jedna rzecz tylko bo przedwczoraj bodajże wszedł PR od Seby i mamy jedną główną zmiana z linkami, i do tej swojej zmiennej mogłabyś dodać, aby wartości w kluczu `to` były ze zmiennej SUB_PAGES, która jest w utils/constants/links. Czyli np. zamiast `/news` to dodać `SUB_PAGES.NEWS`

I tak możesz zrobić dla home, news, joinUs. ( contact na razie możesz tak zostawić bo to nie wiadomo jak jeszcze będzie, a home musisz dodać do SUB_PAGES bo nie ma takiej zmiennej tam )

Jedna uwaga, że joinUs z uwagi na to, że tam będzie link zewnętrzny a nie wewnętrzy to trzeba tam zastosować znacznik `<a>` zamiast `<Link>`. Może to np. ograć za pomocą if wewnątrz `map`. 

Jakby coś było nie jasne to pisz!